src/earn/EarnPoolInfoScreen.test.tsx

```diff
@@ -0,0 +1,131 @@
+import { fireEvent, render, within } from '@testing-library/react-native'
+import React from 'react'
+import { Provider } from 'react-redux'
+import EarnPoolInfoScreen from 'src/earn/EarnPoolInfoScreen'
+import { Screens } from 'src/navigator/Screens'
+import { EarnPosition } from 'src/positions/types'
+import { NetworkId } from 'src/transactions/types'
+import { navigateToURI } from 'src/utils/linking'
+import MockedNavigator from 'test/MockedNavigator'
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+
+const defaultStore = createMockStore({})
+
+const mockPool = {
```

Comment:
nit: can we use mockEarnPositions from test/values

Reply:
Switched to the mockEarnPositions in f1ba98a818495e93db560d400643f0253cf3fc48.